packages/stylelint-plugin/src/util/cssStandardFunctions.ts

```diff
@@ -0,0 +1,111 @@
+/*
+    A set of all function names that are part of the CSS specification.
+    Because they arent (shouldn't be) evaluated at compile-time (apart from some SASS overloaded ones like rgb, rgba), they should support CSS variables within their parameters.
+    Source: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions
+    To generate this list again, go to https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions and run this in your dev console:
+    `Array.from(document.querySelectorAll("#content > article > div:nth-child(22) > div li a")).map(n => n.textContent.replace("()", ""))`
```

Comment:
/nit This feels like it's very prone to break / become out of date with the next update to this MDN doc. IMO the link to the document it enough.

Reply:
Great call out. I thought the same at one point but then forgot.